lib/chef/provider/cron.rb

```diff
@@ -237,7 +237,11 @@ def get_crontab_entry
         newcron = ""
         newcron << "# Chef Name: #{new_resource.name}\n"
         [ :mailto, :path, :shell, :home ].each do |v|
-          newcron << "#{v.to_s.upcase}=#{@new_resource.send(v)}\n" if @new_resource.send(v)
```

Comment:
This isn't caused by you, but for consistency can you use `new_resource` instead of `@new_resource`?